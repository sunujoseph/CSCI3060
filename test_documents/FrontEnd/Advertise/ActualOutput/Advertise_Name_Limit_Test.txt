Enter Username: Enter Item Name: Error: Invalid Name
Enter Item Name: Enter Minimum Bid: Enter Number Of Days Until Auction Ends: Error: Invalid Input
Enter Number Of Days Until Auction Ends: 