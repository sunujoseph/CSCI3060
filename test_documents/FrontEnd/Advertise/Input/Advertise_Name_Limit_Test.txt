login
userSS
advertise
qwertyuioplkjhgfdsazx bnmkhdeqwqvgerwbvhrswrssg
20
50
