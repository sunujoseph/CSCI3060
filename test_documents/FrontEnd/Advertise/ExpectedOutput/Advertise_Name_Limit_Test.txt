Enter Username: 
Enter Item Name: 
Error: Invalid Name
Enter Item Name: 